Makefile

```diff
@@ -48,9 +48,9 @@ OPM ?= $(LOCALBIN)/opm
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.9.1
-CONTROLLER_TOOLS_VERSION ?= v0.8.0
+CONTROLLER_TOOLS_VERSION ?= v0.9.2
 ENVTEST_VERSION ?= latest
-OPERATOR_SDK_VERSION ?= v1.21.0
+OPERATOR_SDK_VERSION ?= v1.25.4
```

Comment:
Any specific reason to use v1.25.4?
v1.27.0 is available

Reply:
v1.25.0 and above supports multi-arch build support so why don't we add a target as suggested in version upgrade docs?

Reply:
v1.25.4 is recommended to be installed according to the docs
https://docs.okd.io/4.12/operators/operator_sdk/golang/osdk-golang-updating-projects.html